apps/src/templates/Notification.jsx

```diff
@@ -107,13 +114,20 @@ const Notification = React.createClass({
     this.setState({open: !this.state.open});
   },
 
+  onAnnouncementClick: function () {
+    if (this.props.analyticId) {
+      trackEvent('teacher_announcement','click', this.props.analyticId);
```

Comment:
I think the more pure React way of doing this would be to pass in trackEvent as a prop.